Library/Formula/tmate.rb

```diff
@@ -0,0 +1,37 @@
+class Tmate < Formula
+  desc "Instant terminal sharing"
+  homepage "http://tmate.io"
+  url "https://github.com/tmate-io/tmate/archive/2.2.0.tar.gz"
+  sha256 "932148b24d9c67e524ce744c9480b5603c9f976afb6ad7ca6d63246cd9e7fd3a"
+
+  head "https://github.com/tmate-io/tmate.git"
+
+  bottle do
+    cellar :any
+    root_url "https://github.com/tmate-io/tmate.io/raw/gh-pages/cellar"
+    sha256 "15cd75a8cab65bd5ede8f329d94b1a561c2f44bb73fbcc25e7dd7474ad42fa90" => :el_capitan
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf"   => :build
+  depends_on "automake"   => :build
+  depends_on "libtool"    => :build
+
+  depends_on "libevent"
+  depends_on "libssh"
+  depends_on "msgpack"
+
+  def install
+    system "sh", "autogen.sh"
+
+    ENV.append "LDFLAGS", "-lresolv"
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/tmate", "-V"
```

Comment:
tmate is a fork of tmux, and --long-options are not implemented. (same as https://github.com/Homebrew/homebrew/blob/master/Library/Formula/tmux.rb#L68)